src/db.py

```diff
@@ -263,3 +263,21 @@ def get_contributors(self, username, submission_name):
         if submission_doc is None:
             raise SubmissionDoesNotExistError(str(username) + ": " + str(submission_name))
         return submission_doc["contributors"]
+
+
+    def get_projects(self, username):
+        project_db = self.mongo.gitsubmit.projects
+        resultCursor = project_db.find({"owner": username})
+        return resultCursor.toArray()
```

Comment:
Oh, one other thing: datetimes in projects need to be converted to a string that the frontend will understand (see https://github.com/gitsubmit/gitsubmit-backend/blob/master/src/db.py#L105 )